packages/embark-solidity/src/index.js

```diff
@@ -26,7 +26,7 @@ class Solidity {
         if (typeof req.body.code !== 'string') {
           return res.send({error: 'Body parameter \'code\' must be a string'});
         }
-        const input = {[req.body.name]: {content: req.body.code.replace(/\r\n/g, '\n')}};
+        const input = {[req.body.name.replace(/\\/g, '/')]: {content: req.body.code.replace(/\r\n/g, '\n')}};
```

Comment:
I'd like to see this done in a helper method that also wraps `path.normalize` to make sure it's all done.

Reply:
Thing is, `path.normalize` would actually give backslahes on Windows, but in this case I specifically want forward slashes. But I can make a helper function called `toForwardSlashes()` if that's what you meant

Reply:
Could maybe do something like
```
path.normalize(...).replace(/\\/g, '/')
```
And maybe call it something along the lines of `normalizeForwardSlash` or possibly just use a `toForwardSlash` parameter in the function, a la:
`packages/embark-utils`:
```
export function normalize (content, useForwardSlashes = true) {
  const result = path.normalize(content);
  if (useForwardSlashes) {
    result = result.replace(/\\/g, '/');
  }
  return result;
}
```

Reply:
Done my dudes!